src/core_plugins/timelion/server/fit_functions/average.js

```diff
@@ -27,7 +27,7 @@ module.exports = function (dataTuples, targetTuples) {
 
     dataTuplesQueue.splice(0, i);
 
-    const sum = _.reduce(avgSet, function (sum, num) { return sum + num; }, 0);
+    const sum = avgSet.reduce((sum, num) => sum + num);
```

Comment:
some timelion CI tests are failing. this is likely due to there line here:

can you add the accumulator:

`avgSet.reduce((sum, num) => sum + num, 0);`

that probably should do it.